### PR TITLE
Inline keyboard

### PR DESCRIPTION
**Pull request by orenyomtov**

### CODE REVIEW

File: telegram_handler.rb [hunk 1]
```diff
@@ -53,55 +65,56 @@ def self.send_message(message:)
...
               if rating >= 0 && rating <= 5
                 Mood::Database.database[:moods].insert({
-                  time: Time.at(message.date),
-                  chat_id: message.chat.id,
+                  time: Time.now,
+                  chat_id: this_chat_id,
                   value: rating
                 })
-                bot.api.send_message(chat_id: message.chat.id, text: "Got it! It's marked in the books 📚")
+                bot.api.send_message(chat_id: this_chat_id, text: "Got it! ("+rating.to_s+") It's marked in the books 📚")
```
Comment (orenyomtov) [inline telegram_handler.rb]:
```suggestion
                bot.api.send_message(chat_id: this_chat_id, text: "Got it ("+rating.to_s+")! It's marked in the books 📚")
```

File: telegram_handler.rb [hunk 2]
```diff
@@ -53,55 +65,56 @@ def self.send_message(message:)
...
               else
-                bot.api.send_message(chat_id: message.chat.id, text: "Only values from 0 to 5 are allowed")
+                bot.api.send_message(chat_id: this_chat_id, text: "Only values from 0 to 5 are allowed")
               end
             else
               self.handle_input(bot, message)
-            end
-          rescue
+            end          
+          #rescue
```
Comment (orenyomtov) [inline telegram_handler.rb]: You removed an important `begin rescue` clause that keeps the bot from crashing when a user blocks the bot (causes `bot.api.send_message` to raise an exception)

File: telegram_handler.rb [hunk 3]
```diff
@@ -115,7 +128,49 @@ def self.handle_input(bot, message)
         #   bot.api.send_message(chat_id: message.chat.id, text: "Number of months tracked: #{number_of_months.round(1)}")
         #   bot.api.send_message(chat_id: message.chat.id, text: "Averaging #{average_number_of_moods.round(1)} per day")
         when "/start"
+          kb=$kb;
```
Comment (orenyomtov) [inline telegram_handler.rb]: Similar keyboard code is repeated multiple times.
How about we encapsulate it in a function that receives a chat_id and returns the keyboard object?

File: telegram_handler.rb [hunk 4]
```diff
@@ -115,7 +128,49 @@ def self.handle_input(bot, message)
...
+          label_content = message.text.split("/setlabel ").last
+          label_mood = label_content[0]
+          label_text = label_content[2,label_content.length - 1]
+          
+          if (label_mood=="0" or label_mood=="1" or label_mood=="2" or label_mood=="3" or label_mood=="4" or label_mood=="5")
+            Mood::Database.database[:moodlabels].replace({
+              chat_id: message.chat.id,
+              mood: label_mood,
+              label: label_text
+            })
```
Comment (orenyomtov) [inline telegram_handler.rb:150]: Else return an error message?

File: telegram_handler.rb [hunk 5]
```diff
@@ -115,7 +128,49 @@ def self.handle_input(bot, message)
...
         when "/start"
+          kb=$kb;
+          Mood::Database.database[:moodlabels].where(:chat_id => message.chat.id).each do |n|
+            _mood = n[:mood].to_i
+            _label = n[:label].to_s
+            kb[5 - _mood] = Telegram::Bot::Types::InlineKeyboardButton.new(text:"#{_mood}: #{_label}", callback_data: _mood)
+          end          
+          answers = Telegram::Bot::Types::InlineKeyboardMarkup.new(inline_keyboard: kb)
+
           bot.api.send_message(chat_id: message.chat.id, reply_markup: answers, text: "🙋‍♂️ Welcome to feelike! 🙋‍♀️\nI will help you keep track of your mood.\nThree times a day I will ask you how do you feel at the moment.\nYou can use my special moods keyboard or just type in a 0-5 number (5 being the happiest).\nWhen you want to see your progress just send me '/graph'\n🦋\nSo let's give it a try! how do you feel like right now?")
```
Comment (orenyomtov) [inline telegram_handler.rb]: Introduce the `/setlabel` command to new users during onboarding

**Review by orenyomtov — COMMENTED** [review]
@michaelhrmn

File: telegram_handler.rb (hunk 4 above)
Comment (michaelhrmn) [reply telegram_handler.rb:150]: There is an else there. Look two lines down

**Pull request closed without merging**